architecture/core_concepts/projects_and_users.adoc

```diff
@@ -138,13 +138,10 @@ xref:../infrastructure_components/web_console.adoc#architecture-infrastructure-c
 [[architecture-projects-projects-at-install]]
 === Projects provided at installation
 
-{product-title} comes with a number of projects out of the box, and `openshift` is the most essential to users:
+{product-title} comes with a number of projects out of the box, and `openshift-*` are the most essential to users:
 
-*`openshift`*
-A user-facing project, mainly for housing objects for day-to-day tasks. These
-include any application objects for access by multiple projects, such as
-templates and images. These objects should be those that do not require
-communication between the pods.
+*`openshift-*`*
+Starting from 3.10 and above, we have a number of projects starting with openshift- to host our master components running as pods and other infrastructure components. The pods created in these namespaces when having a xref:https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/#rescheduler-guaranteed-scheduling-of-critical-add-ons[critical pod annotation], would be considered critical and they would have guaranteed admission by kubelet. Pods created for master components in these namespaces are already marked critical.
```

Comment:
I know that this is kind of vague regarding the number of project, do someone have exact number of projects created with openshift-*?

Reply:
You need to list and describe each of the projects.

Reply:
Unfortunately, I don't have enough knowledge on those areas. The projects that are created are owned by various teams.

Reply:
@ravisantoshgudimetla @kalexand-rh Maybe we could get this list outside of this PR? We should have a list, if we can. This PR is to state that these projects are marked critical. Probably sufficient to move this PR?

Reply:
I think, the project list should go into https://github.com/openshift/openshift-docs/blob/master/architecture/infrastructure_components/kubernetes_infrastructure.adoc